src/MSBuild/TerminalLogger/Project.cs

```diff
@@ -69,4 +73,15 @@ public void AddBuildMessage(MessageSeverity severity, string message)
         BuildMessages ??= new List<BuildMessage>();
         BuildMessages.Add(new BuildMessage(severity, message));
     }
+
+    /// <summary>
+    /// Filters the build messages to only include errors.
```

Comment:
My instinct is to include warnings in the recap, and the console logger does. Do you think it should be errors only?

Reply:
I expected that discussion :) So far, I have implemented it for errors as the original ticket mentions only errors, but I am ready to add warnings as well.

> ...emit an error recap if errors were emitted...